internal/bminventory/inventory.go

```diff
@@ -97,6 +97,7 @@ type Config struct {
 	AgentTimeoutStart               time.Duration     `envconfig:"AGENT_TIMEOUT_START" default:"3m"`
 	ServiceIPs                      string            `envconfig:"SERVICE_IPS" default:""`
 	DeletedUnregisteredAfter        time.Duration     `envconfig:"DELETED_UNREGISTERED_AFTER" default:"168h"`
+	DeregisterInactiveAfter         time.Duration     `envconfig:"DELETED_INACTIVE_AFTER" default:"720h"`
```

Comment:
Why is it defined twice? Is it used in this file? the same for `DeletedUnregisteredAfter`